ipfssync/subpub/message.go

```diff
@@ -26,9 +25,7 @@ func (ps *SubPub) SendMessage(w *bufio.Writer, msg []byte) error {
 
 func (ps *SubPub) ReadMessage(r *bufio.Reader) (*Message, error) {
 	message := new(Message)
-	bare.MaxArrayLength(bareMaxArrayLength)
-	bare.MaxUnmarshalBytes(bareMaxUnmarshalBytes)
```

Comment:
in the end, i wasn't sure about lowering these limits to the default, so in the last rebase i left them as they were (higher), just in case, for example, the pin list grows bigger than 4096 pins. so disregard this comment.